test/e2e/storage/csi_volumes.go

```diff
@@ -223,3 +227,144 @@ func (g *gcePDCSIDriver) cleanupCSIDriver() {
 	csiServiceAccount(cs, config, "gce-controller", true /* teardown */)
 	csiServiceAccount(cs, config, "gce-node", true /* teardown */)
 }
+
+type rbdCSIDriver struct {
+	controllerClusterRoles   []string
+	nodeClusterRoles         []string
+	controllerServiceAccount *v1.ServiceAccount
+	nodeServiceAccount       *v1.ServiceAccount
+	serverPod                *v1.Pod
+	secret                   *v1.Secret
+	serverIP                 string
+
+	f      *framework.Framework
+	config framework.VolumeTestConfig
+}
+
+func initCSIrbd(f *framework.Framework, config framework.VolumeTestConfig) csiTestDriver {
+	return &rbdCSIDriver{
+		nodeClusterRoles: []string{
+			csiDriverRegistrarClusterRoleName,
+		},
+		controllerClusterRoles: []string{
+			csiExternalAttacherClusterRoleName,
+			csiExternalProvisionerClusterRoleName,
+			csiDriverSecretAccessClusterRoleName,
+		},
+
+		f:      f,
+		config: config,
+	}
+}
+
+func (r *rbdCSIDriver) createStorageClassTest(node v1.Node) storageClassTest {
+	return storageClassTest{
+		name:        "csi-rbdplugin",
+		provisioner: "csi-rbdplugin",
+		parameters: map[string]string{
+			"monitors": r.serverIP,
+			"pool":     "rbd",
+			"csiProvisionerSecretName":      r.secret.Name,
+			"csiProvisionerSecretNamespace": r.config.Namespace,
+		},
+		claimSize:          "1Gi",
+		expectedSize:       "1Gi",
+		nodeName:           node.Name,
+		skipWriteReadCheck: true,
```

Comment:
why not test r/w?

Reply:
If skipWriteReadCheck is false, both rbd and cephfs tests fail in second 
[```runInPodWithVolume```](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/storage/volume_provisioning.go#L153) run. I will file another issues for csi cephfs and rbd.

Until the issues are fixed, is it O.K. to merge this PR with ```skipWriteReadCheck=true```?

Reply:
thanks!